src/Rules/Resources/HowToFix.resx

```diff
@@ -544,7 +539,7 @@ If possible, use a predefined (non-custom) control type and the default localize
     <value>If the sibling hyperlink with the same Name navigates to a different destination, change the Name of either hyperlink.</value>
   </data>
   <data name="PropertyExcludesPrivateUnicodeCharacters" xml:space="preserve">
```

Comment:
Similar to my other comment, it seems like this string covers the `LocalizedLandmarkTypeExcludesPrivateUnicodeCharacters` string added above